### PR TITLE
backup: add backup ddl to meta

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

Support to backup DDLs to BackupMeta.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module github.com/pingcap/kvproto
 
 require (
-	github.com/gogo/protobuf v1.3.1
+	github.com/gogo/protobuf v0.0.0-20180717141946-636bf0302bc9
```
Comment (kennytm) [inline go.mod]: Why?

**Comment by MyonKeminta** [comment]
Is it necessary to update protobuf version?

**Comment by 5kbpers** [comment]
> Is it necessary to update protobuf version?

I use a different version of protoc and that updates it. Did not notice pushed the `go.mod`. 😓

**Comment by MyonKeminta** [comment]
Emmm.... I think if the protoc version is correct, there should not be so much changes in go files...

File: proto/backup.proto [hunk 2]
```diff
@@ -40,6 +40,9 @@ message BackupMeta {
     // backup data's range is represented by raw_ranges.
     bool is_raw_kv = 8;
     repeated RawRange raw_ranges = 9;
+
+    // In incremental backup, DDLs which are completed in (lastBackupTS, backupTS] will be stored here.
+    bytes ddls = 10;
```
Comment (overvenus) [inline proto/backup.proto]: What's the actual format of these DDLs? And how about `repeated bytes ddls = 10`

Comment (5kbpers) [reply proto/backup.proto]: it is an array of `model.Job` that unmarshalled in JSON format, I think `bytes` is good enough.

**Comment by 5kbpers** [comment]
PTAL

**Comment by kennytm** [comment]
Please revert all changes to `*.pb.go` except `backup.pb.go`.

**Comment by 5kbpers** [comment]
> LGTM, but if there's a solution to pass the jobs structurally instead of JSON it would be even better.

What we need to store is `model.Job` so I think JSON is more convenient. 😅

**Pull request merged**